README.md

```diff
@@ -33,7 +33,19 @@ To test the project, you need first to install the server and get it running, th
 To install and start the server:
 
 1. Install Docker.
-   - [On Windows](https://docs.docker.com/docker-for-windows/install/).
+   - [On Windows 10 Pro](https://docs.docker.com/docker-for-windows/install/).
+   - [On Windows 10 Home and below](https://github.com/docker/toolbox/releases).
+      - Addition steps for Docker Toolbox(Windows 10 Home and below)
+         - Run the Docker Quickstart Terminal shortcut
```

Comment:
What is the `Docker Quickstart Terminal shortcut`? Install instructions should be as explicit as possible. I think this would confuse or trip up some users.

Reply:
Upon installing "docker toolbox" it will create a few shortcuts. Two of them are
"Oracle VM VirtualBox" and "Docker Quickstart Terminal"

Reply:
Then I would write it as

> Double-click on the Docker Quickstart Terminal shortcut created by the docker toolbox installer.